plugins/org.python.pydev.core/src/org/python/pydev/core/ModulesKeyForFolder.java

```diff
@@ -0,0 +1,44 @@
+package org.python.pydev.core;
+
+import java.io.File;
+
+import org.python.pydev.shared_core.string.FastStringBuffer;
+
+public class ModulesKeyForFolder extends ModulesKey {
+    /**
+     * 1L = just name and file
+     * 2L = + zipModulePath
+     * 3L = + folderModulePath
+     */
+    private static final long serialVersionUID = 3L;
+
+    public String folderModulePath;
```

Comment:
I don't think you need the path in this case (just the name and file should suffice... I guess you just got it from the zip version where the path is needed, but that's not true in this case).